src/assets/toolkit/styles/templates/_print.scss

```diff
@@ -74,11 +74,22 @@
 .hide-for-print {
   @media print {
     display: none;
-  }  
+  }
 }
 
 .show-for-print {
   @media print {
     display: block;
-  }  
+  }
+}
+
+@media print {
+  [ng\:cloak], [ng-cloak], [data-ng-cloak], [x-ng-cloak], .ng-cloak, .x-ng-cloak, .ng-hide:not(.ng-hide-animate) {
+   display: block !important;
```

Comment:
@kaptron Hm I'm a little confused 😛  If we update `ng-text-truncate`, and remove

```
+@media print {
+  [ng\:cloak], [ng-cloak], [data-ng-cloak], [x-ng-cloak], .ng-cloak, .x-ng-cloak, .ng-hide:not(.ng-hide-animate) {
+   display: block !important;
```

the `ng-truncate` stuff will show for print, but the manual stuff won't, because we need the code above for our manual stuff to show on print. Adding `.hide-for-print` and `.show-for-print` didn't work.